Essentials/src/main/java/com/earth2me/essentials/Essentials.java

```diff
@@ -144,6 +146,7 @@
     private transient PotionMetaProvider potionMetaProvider;
     private transient ServerStateProvider serverStateProvider;
     private transient ContainerProvider containerProvider;
+    private transient SerializationProvider serializationProvider = null;
```

Comment:
A non-primitive is by default `null`; this breaks the style around unnecessarily.

Reply:
```suggestion
    private transient SerializationProvider serializationProvider;
```